src/Uno.Extensions.Logging/HostBuilderExtensions.cs

```diff
@@ -6,13 +6,14 @@ public static class HostBuilderExtensions
 {
 	public static IHostBuilder UseLogging(
 		this IHostBuilder hostBuilder,
-		Action<ILoggingBuilder> configure)
+		Action<ILoggingBuilder> configure, bool enableUnoLogging = false)
```

Comment:
This is what we do for navigation:
https://github.com/unoplatform/uno.extensions/blob/081dad60b5d5f98906a6782f3ea687aa75100b5d/src/Uno.Extensions.Navigation.UI/HostBuilderExtensions.cs#L10
We have a NavigationConfig function callback - I would suggest we try to maintain consistency with this.